### PR TITLE
Add 'container' config to READMEs

### LINKED ISSUE #422: Update docs with 'container' configuration
https://github.com/GoogleContainerTools/jib/pull/421

### PR DESCRIPTION
**Pull request by TadCordle**
Fixes #422. Merge after 0.9.1 release.

### CODE REVIEW

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -174,13 +171,22 @@ Property | Type
...
+<a name="container-closure"></a>`container` is a closure with the following properties:
+
+Property | Type | Default | Description
+--- | --- | --- | ---
+`jvmFlags` | `List<String>` | *None* | Additional flags to pass into the JVM when running your application.
+`mainClass` | `String` | *Inferred\** | The main class to launch your application from.
+`args` | `List<String>` | *None* | Default main method arguments to run your application with.
+`format` | `String` | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
+
+
```
Comment (coollog) [inline jib-gradle-plugin/README.md]: nit: extra newline?

File: jib-gradle-plugin/README.md [hunk 2]
```diff
@@ -174,13 +171,22 @@ Property | Type
...
 ### Example
 
 In this configuration, the image is:
 * Built from a base of `openjdk:alpine` (pulled from Docker Hub)
 * Pushed to `localhost:5000/my-image:built-with-jib`
-* Runs by calling `java -Xms512m -Xdebug -Xmy:flag=jib-rules -cp app/libs/*:app/resources:app/classes mypackage.MyApp some args`
-* Reproducible
```
Comment (coollog) [inline jib-gradle-plugin/README.md]: Why was this removed?

Comment (TadCordle) [reply jib-gradle-plugin/README.md]: Images are reproducible regardless of configuration, so it seemed out of place since this is a configuration-focused example.

Comment (coollog) [reply jib-gradle-plugin/README.md]: Okay that makes sense.
